sdks/python/apache_beam/typehints/typehints_test.py

```diff
@@ -918,139 +776,16 @@ def test_must_be_single_return_type(self):
       def unused_foo():
         return 4, 'f'
 
-  def test_type_check_violation(self):
-    @check_type_hints
-    @with_output_types(int)
-    def foo(a):
-      return 'test'
-    with self.assertRaises(TypeCheckError) as e:
-      m = 4
-      foo(m)
-
-    self.assertEqual("Type-hint for return type violated. Expected an "
-                     "instance of {}, instead found an instance of "
-                     "{}.".format(int, type('test')),
-                     e.exception.args[0])
-
-  def test_type_check_simple_type(self):
-    @with_output_types(str)
-    def upper(a):
-      return a.upper()
-    self.assertEqual('TEST', upper('test'))
-
   def test_type_check_composite_type(self):
     @with_output_types(typehints.List[typehints.Tuple[int, int]])
-    def bar():
+    def unused_bar():
```

Comment:
What exactly is this testing?

Reply:
Not sure. Maybe the original author intended to add the `@check_type_hints` decorator to the function? (same for `test_any_return_type_hint` below)

Reply:
Or more likely it's verifying that a composite type is accepted as a type hint.